packages/ui-components/src/components/status-box-component.ts

```diff
@@ -0,0 +1,39 @@
+import { html } from 'lit';
+import { BaseElement } from '../base';
+import { property } from 'lit/decorators.js';
+
+export class StatusBoxComponent extends BaseElement {
+  @property({ type: Array })
+  boxes: { value: string; label: string; color: string }[] = [
+    { value: '1500+', label: 'projects onboarded', color: 'text-yellow-400' },
+    { value: '78%', label: 'average score', color: 'text-green-600' },
+    { value: '250', label: 'GitHub stars', color: 'text-yellow-400' },
+  ];
```

Comment:
Colors should be predefined, probably somewhere else in this file (to help tailwind).

Reply:
Also I think this component should be split, preferably just as a single component (atomic design).